tracing-attributes/tests/instrument.rs

```diff
@@ -238,3 +238,32 @@ fn impl_trait_return_type() {
 
     handle.assert_finished();
 }
+
+#[test]
+fn keywords_in_fields() {
```

Comment:
Could we please get another test for the negative case where this error gets triggered:

https://github.com/tokio-rs/tracing/pull/2606/files#diff-32188590fd36b1fe3eb8706535c1ec7bc9c046e27d4c540b5eeba49e47724d26R367-R369